recipe/meta.yaml

```diff
@@ -12,11 +12,11 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vvv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vv
   missing_dso_whitelist:  # [win]
    - $RPATH/cudart64_101.dll  # [win]
-  skip: true # [s390x]
+  skip: true # [py<37 or s390x]
```

Comment:
must we still skip eol versions?

Reply:
I think we should. It does not hurt but is then compliant with the upstream and adds another bit of info and the rule should we try to rebuild it for older py version for some unexpected reason.